herald/src/app/shared/realm.enum.ts

```diff
@@ -1,5 +1,5 @@
 export const enum Realm {
-  Albion,
-  Hibernia,
-  Midgard
+  Albion = 1,
```

Comment:
True that, we really just need the mapping in the service code. I'll push a change as additional commit. We can squash it in the end.